src/ui/reference/_colors.scss

```
@@ -1,11 +1,34 @@
 /* stylelint-disable color-no-hex */
-$black: black;
-$dark-grey: #484848;
-$grey: #757575;
-$light-grey: #EBEBEB;
 
-$red: #A43030;
+:root {
+  // Button
+  --button-bg: black;
 
-$default-box-shadow: 0 0 5px $light-grey;
+  // Dropdown
+  --dropdown-text-color: black;
+  --dropdown-header-button-bg: #ebebeb;
```

I was thinking maybe we can still have variables for colors.

```
$light-grey: #ebebeb;
```
and then 
```suggestion
  --dropdown-header-button-bg: $light-grey;
```